src/Equinor.ProCoSys.IPO.Command/Email/Smtp.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace Equinor.ProCoSys.IPO.Command.Email
+{
+    public class Smtp
```

Comment:
Maybe SmtpConfig or something similar to better express the intent of the class?

Reply:
SmtpOptions